ivadomed/postprocessing.py

```diff
@@ -137,3 +138,47 @@ def mask_predictions(predictions, mask_binary):
     assert predictions.shape == mask_binary.shape
     assert np.array_equal(mask_binary, mask_binary.astype(bool))
     return predictions * mask_binary
+
+
+def coordinate_from_heatmap(nifti_image, thresh=0.3):
+    """
+    Retrieve coordinates of local maxima in a soft segmentation.
+    Args:
+        nifti_image (nibabel object): nifti image of the soft output.
+        thresh (float): Relative threshold for local maxima, i.e., after normalizing
+        the min and max between 0 and 1, respectively.
+
+    Returns:
+        list: A list of computed coordinates found by local maximum. each element will be a list composed of
+        [x, y, z]
+    """
+    image_ref = nib.load(nifti_image)
+    image = np.array(image_ref.dataobj)
+    coordinates_tmp = peak_local_max(image, min_distance=5, threshold_rel=thresh)
+    return coordinates_tmp
+
+
+def label_file_from_coordinates(fname_image, coord_list):
```

Comment:
Just for consistency purposes, what about input a nifti object as the other functions?

Reply:
The thing is, It kind of goes with the previous function. Therefore there is a second option: compact them both into a function that has an argument that determine if the output is coordinate list or a file with single-voxel label what do you think is the best ?

Reply:
In order to reduce the number of I/O --> what about loading the image outside of these functions and only deal with nibabel objects for the I/O of these functions.
But I may miss something: couldyou please help me?